src/frontends/paddle/src/frontend.cpp

```diff
@@ -392,5 +393,9 @@ PADDLE_C_API void* GetFrontEndData() {
     res->m_creator = []() {
         return std::make_shared<ov::frontend::paddle::FrontEnd>();
     };
+#ifndef OPENVINO_DEBUG_ENABLE
+    // disable protobuf logging
+    google::protobuf::SetLogHandler(nullptr);
```

Comment:
google::protobuf::SetLogHandler is not thread-safe. I expect the GetFrontEndData call does not cause race conditions.